src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationUseStyleCheckExamplesTest.java

```diff
@@ -34,36 +38,46 @@ protected String getPackageLocation() {
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-
+            "25:1: " + getCheckMessage(MSG_KEY_ANNOTATION_PARENS_PRESENT),
+            "27:46: " + getCheckMessage(MSG_KEY_ANNOTATION_TRAILING_COMMA_PRESENT),
         };
 
-        verifyWithInlineConfigParser(getPath("Example1.txt"), expected);
+        verifyWithInlineConfigParser(getPath("ExampleAnnotationStyle1.java"), expected);
```

Comment:
We did this format of ExampleX on purpose.
Please share error message that forcing you to do this update.

We need to update xdoc template file to reference new file.

Reply:
> We did this format of ExampleX on purpose. Please share error message that forcing you to do this update.
> 
> We need to update xdoc template file to reference new file.

When I have a classs named Example1 after converting the txt to java, i get the following error:
`Duplicate class found in the file '/Users/piyushkumarsadangi/Development/new/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleDiffer`entStyles.java'

Reply:
https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleDifferentStyles.java

Please rename in this existing java file all Example1, Example2 ... to ExampleA, ExampleB yo avoid conflicts in names. 

Files that you are creating should be Example1, Example2, ...

Reply:
done